libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetryInterface.kt

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.navigation.core.telemetry
+
+import com.mapbox.navigation.core.telemetry.telemetryevents.TelemetryUserFeedback
+
+interface MapboxNavigationTelemetryInterface {
+    fun postUserFeedbackEvent(
```

Comment:
Why do we have a `MapboxNavigationTelemetryInterface` exposing only a function for concrete feedback events?